buildutils/src/create-package.ts

```diff
@@ -37,5 +37,5 @@ inquirer.prompt(questions).then(answers => {
   data.name = name;
   data.description = description;
   utils.writePackageData(jsonPath, data);
-  utils.run('jlpm run integrity');
+  utils.run('npm run integrity');
```

Comment:
Can you comment on the places where `npm` must still be used? I have found that confusing (it looks like it is most of the scripts in buildutils?).